include/alpaka/event/EventUniformCudaHipRt.hpp

```diff
@@ -74,10 +74,15 @@ namespace alpaka
                 // -> No need to synchronize here.
                 ALPAKA_UNIFORM_CUDA_HIP_RT_CHECK(ALPAKA_API_PREFIX(EventDestroy)(m_UniformCudaHipEvent));
             }
+            auto getNativeHandle() const noexcept
```

Comment:
```suggestion

            [[nodiscard]] auto getNativeHandle() const noexcept
```

Note the extra linebreak.

Reply:
I quickly checked clang-format and `SeparateDefinitionBlocks` in clang-format-14 might solve this in the future.